### DISCUSSION TITLE
Unknown Terminal Type in Multipass shell

### DISCUSSION THREAD

**Discussion by anyon17** [ghostty-org/ghostty#4158] [Q&A] [answered]
On my mac I am doing `multipass shell dev-vm` through the ghostty terminal. When I try to `clear` it says
```
xterm-ghostty': unknown terminal type.
```
Also when inside a git repo when I do git branch, it says

```
WARNING: terminal is not fully functional
Press RETURN to continue
```

**Comment by m1keil** [comment, marked as answer]
this is probably terminfo related. see https://ghostty.org/docs/help/terminfo

**Comment by slhck** [comment]
Duplicate: https://github.com/ghostty-org/ghostty/discussions/4077
